package.json

```diff
@@ -82,9 +86,11 @@
     "webpack-merge": "^4.2.1"
   },
   "dependencies": {
-    "@edx/paragon": "^3.8.0",
+    "query-string": "^5.1.1"
+  },
+  "peerDependencies": {
+    "@edx/paragon": "^4.2.4",
     "prop-types": "^15.5.10",
-    "query-string": "^5.1.1",
```

Comment:
Was query-string meant to be a peer dependency?  Could this have stayed and also moved from `dependencies` to `devDependencies`?

Reply:
The diff is a bit wonky here. In the current version, these are all dependencies. This commit moves everything, but `query-string` to peerDependencies.